contracts/digg-core/UFragments.sol

```diff
@@ -45,6 +47,8 @@ contract UFragments is ERC20Detailed, Ownable {
     address public monetaryPolicy;
     uint256 public rebaseStartTime;
 
+    bool private remDiggMint = false;
```

Comment:
When dealing with proxies, I'd stay away from the pattern of initializing storage variables during declaration. This only sets the value in the logic contract and not on the proxy (see https://docs.openzeppelin.com/upgrades-plugins/1.x/proxies#the-constructor-caveat)
```suggestion
    bool private remDiggMint;
```